docs/docsite/rst/dev_guide/module_lifecycle.rst

```diff
@@ -1,50 +1,126 @@
 .. _module_lifecycle:
 
-**********************************
-The lifecycle of an Ansible module
-**********************************
+********************************************
+The lifecycle of an Ansible module or plugin
+********************************************
 
-Modules in the main Ansible repo have a defined life cycle, from first introduction to final removal. The module life cycle is tied to the `Ansible release cycle <release_cycle>`.
-A module may move through these four states:
+Modules and plugins in the main Ansible repository have a defined life cycle, from first introduction to final removal. The module and plugin life cycle is tied to the `Ansible release cycle <release_cycle>`.
+A module or plugin may move through these four states:
 
-1. When a module is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
+1. When a module or plugin is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
```

Comment:
```suggestion
1. When a module or plugin is first accepted into Ansible, is considered as tech preview and will mark it as such in the documentation.
```

Reply:
Shouldn't it be `it is considered` and not `is considered`?

Reply:
I prefer active voice when possible. But does the progression from "tech preview" to "stable" even happen any more in the collections ecosystem?

Reply:
From my understanding, the progression from "tech preview" to "stable" happens in a different way. Like "community.aws" is tech preview when some module matures it is promoted to "amazon.aws" (stable).

Reply:
But right now, there's no way to mark it in a uniform way. (I guess https://github.com/ansible/ansible/pull/73707 will allow, but that is only available for ansible-core 2.12, and not for 2.11 and 2.10.)

Reply:
@acozine @Akasurde how should we proceed here? I'm for keeping that part as-is in this PR. This whole section needs to be rewritten anyway, but since it is not clear what it should say, I would really prefer not to open that can of worms here :)